torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -80,24 +80,27 @@ bool AliasDb::hasWriters(const Value* v) const {
     return false;
   }
 
-  if (isWriteCacheStale_) {
-    rebuildWriteCache();
+  const auto& memoryLocations = elementMap_.at(v)->getMemoryLocations();
+  for (const auto& pr : writeIndex_) {
```

Comment:
This seems expensive, was it this way before?

Reply:
We used to cache the whole set of written locations when comparisons were more inefficient. I can add that back.